src/molgenis-vue-test-utils.js

```diff
@@ -69,9 +69,7 @@ const testAction = (action, options, done) => {
 
     try {
       expect(mutation.type).to.equal(type)
-      if (payload) {
-        expect(mutation.payload).to.deep.equal(payload)
-      }
+      expect(mutation.payload).to.deep.equal(payload)
```

Comment:
The helper is taken from the ['official' vuex documentation](https://vuex.vuejs.org/guide/testing.html), i suggest to submit a PR to the vuex docs, as i am not clear on whether this is a bug or a feature.

Also i suggest looking at the vuex suggestion for future use 

> If you have spies available in your testing environment (for example via Sinon.JS), you can use them instead of the testAction helper:

And [vue-test-utils guide](https://vue-test-utils.vuejs.org/guides/using-with-vuex.html) on testing vuex